lib/plugins/prometheus/PrometheusCounterTransactionManager.ts

```diff
@@ -0,0 +1,59 @@
+import type { TransactionObservabilityManager } from '@lokalise/node-core'
+import type { IFastifyMetrics } from 'fastify-metrics'
+import type { Counter } from 'prom-client'
+
+/**
+ * TransactionObservabilityManager implementation that uses Prometheus counter
```

Comment:
can we mention it in the readme?

Reply:
Absolutely! will do it later or tomorrow morning :D